### ISSUE TITLE
windws.h {linux}

### ISSUE THREAD

**Issue by Yaksinikos** [hexabits/nifskope#12]
I've tried to build in linux, but widnows.h is missing there -.-
how did you get your build; becasue if I leave bsatest I get missing issues in message.h

**Comment by psi29a** [comment]
This should be fixed in this commit:
https://github.com/niftools/nifskope/pull/101

Please test this branch. I used the following:

```
/usr/lib/x86_64-linux-gnu/qt5/bin/qmake -makefile NifSkope.pro
```

**Comment by Yaksinikos** [comment]
@psi29a nope, do not work for me :& same error (using `dev` branch, `psi29a:2.0.0-dev6-linux-fix`"

**Comment by psi29a** [comment]
@Translator5 How is that? windows.h was removed in headers so it will be impossible to fail in the same way.

See for yourself: 
https://github.com/niftools/nifskope/pull/101/files#diff-583527b78fce7a9dd0c28359629642ff

Line 15

Please post your error.

Result of my branch:
![Image of Nifscope 2.0](https://dl.dropboxusercontent.com/u/396161/Screenshot%20from%202017-01-13%2020-31-18.png)

**Comment by Yaksinikos** [comment]
@jonwd7 
> Please post your error.

```
Project MESSAGE: lupdate could not be found, ignoring make target
Project MESSAGE: lrelease could not be found, ignoring build step
```

**Comment by psi29a** [comment]
Those messages don't have anything to do with not having windows.h. I used to get those too, but that was because I didn't initialize the submodules such as zlib, docsys, kfmxml, nifxml  and qhull.

```
git clone --recursive git://github.com/psi29a/nifskope.git
git checkout 2.0.0-dev6-linux-fix
/usr/lib/x86_64-linux-gnu/qt5/bin/qmake -makefile NifSkope.pro
make -j4
```

I just tested, from scratch. :)

**Comment by psi29a** [comment]
Works for you now @Translator5 ?

**Comment by Yaksinikos** [comment]
@psi29a nope, I dont get it to run :/ hmm may some commands run from false directory or sth. like that ... :confused:

**Comment by psi29a** [comment]
@Translator5 : what distribution are you using?
Also, with the commands I posted above, can you use pastebin or gist to post the output of all the commands and post the link here?  I would really like to solve this mystery. :)
